tools/xchain-suite/src/components/swap/SwapTrackingModal.tsx

```
@@ -250,7 +250,7 @@ export function SwapTrackingModal({
   const trackerUrl = protocol === 'Chainflip'
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)